pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -183,10 +185,15 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 		PullSecret:   installConfig.PullSecret,
 		ReleaseImage: releaseImage,
 		EtcdCluster:  strings.Join(etcdEndpoints, ","),
+		Proxy:        installConfig.Proxy,
```

Comment:
This should not be the install-config, but the status from object from https://github.com/openshift/installer/pull/1866